packages/rest_api/build.yaml

```
@@ -0,0 +1,10 @@
+global_options:
+  dart_mappable_builder:
+    runs_before:
+      - retrofit_generator
+    options:
+      discriminatorKey: type
+      ignoreNull: true
+      renameMethods:
+        toJson: toJsonString
+        toMap: toJson
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)